### PR TITLE
raftstore: add applying snap count

### PR DESCRIPTION
**Pull request by BusyJay**
ref tikv/proto#115

@siddontang @huachaohuang PTAL

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -64,6 +64,7 @@ git = "https://github.com/ngaut/rust-rocksdb.git"
 
 [dependencies.kvproto]
 git = "https://github.com/pingcap/kvproto"
+branch = "busyjay/apply-snapshot-count"
```
Comment (huachaohuang) [inline Cargo.toml]: Shouldn't we use master?

**Comment by huachaohuang** [comment]
LGTM

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -1392,6 +1392,17 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         STORE_SNAPSHOT_TRAFFIC_GAUGE_VEC.with_label_values(&["receiving"])
             .set(snap_stats.receiving_count as f64);
 
+        let mut apply_snapshot_count = 0;
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: any test to cover?

Comment (BusyJay) [inline src/raftstore/store/store.rs]: It's hard to control the applying process. Snapshot is probably applied before store heartbeat is about to be collected.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**